README.md

```diff
@@ -1,2 +1,41 @@
 # Portfolio
 My personal portfolio
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> This is My personal portfolio project, I'm starting off with HTML and CSS and I plan to add more features to this repository in the future and eventually use it as my main portfolio!
+
+## Built With
+
+- Html & Css
+
+## Authors
+
```

Comment:
- [ ] Please since you are the only contributor of this project you can singularize the `Authors` section to `Author` :pray: